examples/gke/gke_cluster_autoscaler/gke.tf

```diff
@@ -0,0 +1,36 @@
+# 2. Create GKE cluster.
+
+module "gke" {
+  source                     = "terraform-google-modules/kubernetes-engine/google"
+  version                    = "24.1.0"
+  project_id                 = var.project_id
+  name                       = var.cluster_name
+  region                     = var.cluster_region
+  zones                      = var.cluster_zones
+  network                    = module.vpc.network_name
+  subnetwork                 = module.vpc.subnets_names[0]
+  ip_range_pods              = local.ip_range_pods
+  ip_range_services          = local.ip_range_services
+  http_load_balancing        = false
+  network_policy             = false
+  horizontal_pod_autoscaling = true
+  filestore_csi_driver       = false
+
+  node_pools = [
+    {
+      name               = "default-node-pool"
+      machine_type       = "e2-medium"
+      min_count          = 2 # has to be >=2 to successfully deploy CAST AI controller.
+      max_count          = 10
+      autoscaling        = true
```

Comment:
We should disable autoscaling for default GKE node pool

Reply:
Disabled for now.